datafusion/sql/src/expr/function.rs

```diff
@@ -102,6 +103,21 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
                 // Numeric literals in window function ORDER BY are treated as constants
                 false,
             )?;
+
+            let func_deps = schema.functional_dependencies();
```

Comment:
Could you document what ordering is stable means ?

Reply:
Sure, updated comment